### PR TITLE
Cocoapods setup with demo project added.

### PR DESCRIPTION
**Pull request by purvesh-d**
Changes of cocoapods setup and demo project added.

### CODE REVIEW

File: SSVerticalPanoramaImage/Classes/OpenCVClass/CVWrapper.mm [hunk 1]
```diff
@@ -48,6 +48,7 @@ + (nullable UIImage*) processWithImages:(NSArray<UIImage*>*)images error:(NSErro
     try {
         stitchedMat = stitch (matImages);
         UIImage* result =  [UIImage imageWithCVMat:stitchedMat];
+        NSLog (@"result... %@", result);
```
Comment (iOSDev-Auction) [inline SSVerticalPanoramaImage/Classes/OpenCVClass/CVWrapper.mm:51]: Coding Standaard - Print statement

File: SSVerticalPanoramaImage.podspec [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'SSVerticalPanoramaImage'
+  s.version          = '1.0.0'
+  s.summary          = 'A short description of SSVerticalPanoramaImage.'
```
Comment (bhargavbajani-simformsolutions) [inline SSVerticalPanoramaImage.podspec]: missing description

File: SSVerticalPanoramaImage.podspec [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  s.description      = <<-DESC
+                          TODO: Add long description of the pod here.
```
Comment (bhargavbajani-simformsolutions) [inline SSVerticalPanoramaImage.podspec]: Missing description

File: SSVerticalPanoramaImage/Classes/SSVerticalPanorama.swift [hunk 4]
```diff
@@ -9,17 +9,23 @@
...
+        
+    }
+    
+    public func openSSVerticalPanoramaCam(navController: UINavigationController) {
         AVCaptureDevice.requestAccess(for: .video) { success in
             if success {
                 // if request is granted (success is true)
                 DispatchQueue.main.async {
-                    guard let ssCustomeCameraVC = UIStoryboard(name: StoryBoradEnum.ssStoryBoard.rawValue, bundle: nil).instantiateViewController(withIdentifier: ViewContrllerEnum.ssVerticalPanoController.rawValue) as? SSVerticalPanoController else {return}
+                    let bundle = Bundle(for: Self.self)
```
Comment (bhargavbajani-simformsolutions) [inline SSVerticalPanoramaImage/Classes/SSVerticalPanorama.swift:28]: weak self missing

**Review by bhargavbajani-simformsolutions — CHANGES_REQUESTED** [review]

File: SSVerticalPanoramaImage.podspec [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'SSVerticalPanoramaImage'
+  s.version          = '1.0.0'
+  s.summary          = 'Capture stunning vertical panorama images with ease and preview them instantly on the built-in screen.'
+
+  s.description      = <<-DESC
```
Comment (bhargavbajani-simformsolutions) [inline SSVerticalPanoramaImage.podspec:14]: The description title and end  "DESC" should be removed.

Comment (purvesh-d) [reply SSVerticalPanoramaImage.podspec:14]: As I can see in pod official doc syntax mention at [here](https://guides.cocoapods.org/syntax/podspec.html#description),  This "<<-DESC ..... DESC" is the syntax and we have to write our content inside it. 


Also I have explored some well-known pod listed below and they are also doing the same. 

Firebase: https://github.com/firebase/firebase-ios-sdk/blob/master/Firebase.podspec
Kingfisher: https://github.com/onevcat/Kingfisher/blob/master/Kingfisher.podspec
Pod Official Doc: https://guides.cocoapods.org/syntax/podspec.html#description

**Review by bhargavbajani-simformsolutions — APPROVED** [review]

**Pull request merged**